include/staff/templates/task-view.tmpl.php

```diff
@@ -130,10 +130,9 @@ class="icon-reorder"></i> <?php
                 class="action-dropdown anchor-right">
                 <ul>
             <?php foreach ($actions as $a => $action) { ?>
-                    <li>
-                        <a class="no-pjax <?php
-                            echo $action['class'] ?: 'task-action'; ?>"
-                            <?php
+                    <li class="<?php
```

Comment:
Would this be cleaner:
`<?php
                if ($action['class'])
                    echo sprintf("class='%s'", $action['class']); ?> >`